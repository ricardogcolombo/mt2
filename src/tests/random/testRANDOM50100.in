50 100
1 21 42 48 126
1 25 2 33 12
1 41 5 8 124
1 47 103 17 17
1 49 45 14 84
1 5 49 33 56
1 1 44 44 95
1 31 32 47 130
1 7 98 12 9
1 9 126 21 66
1 41 111 19 7
1 46 86 16 37
1 12 16 14 68
1 41 130 4 119
1 24 64 29 7
1 6 83 17 50
1 23 89 47 32
1 41 50 29 93
1 27 124 8 92
1 41 80 43 71
1 27 22 9 109
1 38 90 29 17
1 18 94 7 3
1 20 41 18 26
1 50 37 40 12
1 50 8 39 31
1 26 109 22 90
1 37 58 17 85
1 13 80 19 30
1 45 31 7 125
1 7 63 23 48
1 13 119 10 7
1 25 130 33 22
1 26 15 9 3
1 45 61 17 57
1 38 29 47 56
1 7 71 9 106
1 44 106 33 59
1 32 82 12 105
1 24 123 14 30
1 16 112 34 32
1 40 130 30 86
1 20 108 36 67
1 1 120 46 87
1 48 10 49 31
1 20 21 40 95
1 50 67 45 90
1 36 108 30 105
1 45 9 13 101
1 19 56 46 47
1 50 41 23 49
1 21 22 14 45
1 49 17 27 79
1 1 1 3 77
1 18 38 3 19
1 43 13 10 18
1 44 120 36 6
1 8 50 11 56
1 17 15 45 40
1 12 44 9 88
1 12 108 46 105
1 9 93 14 19
1 49 53 1 34
1 32 112 3 43
1 36 26 27 77
1 12 52 15 33
1 40 31 39 28
1 33 102 13 117
1 13 2 34 130
1 19 92 30 67
1 35 52 49 5
1 7 41 2 48
1 9 85 39 51
1 39 127 28 37
1 42 108 24 27
1 49 110 25 48
1 24 53 48 12
1 37 117 31 51
1 39 118 24 20
1 50 45 21 88
1 7 38 46 62
1 37 93 27 15
1 41 83 11 122
1 43 81 48 99
1 16 91 8 62
1 5 124 12 37
1 5 29 6 81
1 35 65 50 16
1 34 62 50 80
1 29 19 49 42
1 14 16 37 80
1 43 62 47 35
1 11 92 46 100
1 17 17 23 78
1 36 108 28 27
1 28 83 2 115
1 32 94 19 22
1 28 52 32 75
1 19 7 5 88
1 30 120 20 8
