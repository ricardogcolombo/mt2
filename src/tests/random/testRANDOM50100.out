0.363518
0.682988
0.404724
0.300552
0.501617
0.703695
0.521772
0.351298
0.826926
0.498584
0.546291
0.476307
0.682867
0.555681
0.325436
0.550373
0.313163
0.583274
0.369747
0.489874
0.371814
0.438811
0.618234
0.592742
0.394203
0.816434
0.609469
0.210178
0.333888
0.316513
0.280982
0.576826
0.459952
0.422066
0.732050
0.410401
0.626549
0.457817
0.590856
0.629055
0.401656
0.697581
0.311468
0.746774
0.529722
0.478501
0.497381
0.471189
0.551111
0.377089
